python/lsst/jointcal/jointcal.py

```diff
@@ -184,7 +191,7 @@ def run(self, ref, tract):
         andConfig = AstrometryNetDataConfig()
         andConfigPath = os.path.join(anDir, "andConfig.py")
         if not os.path.exists(andConfigPath):
-            raise RuntimeError("astrometry_net_data config file \"%s\" required but not found" %andConfigPath)
+            raise RuntimeError("astrometry_net_data config file \"%s\" required but not found"%andConfigPath)
```

Comment:
Too picky, but I'm curious why you removed the space in front of `%` instead of putting one on both sides of it? Line too long?